src/assignments/hello-world.rst

```diff
@@ -1,4 +1,65 @@
-.. _hello-world:
+.. _assignment0:
 
 Assignment #0: Hello, World!
-============================
\ No newline at end of file
+============================
+
+The purpose of this initial assignment is to familiarize yourself with the process of running the autograding tests and submitting your work via GitHub Classroom. Even if you are familiar with GitHub Classroom, this assignment will contain several new items that are specific to working with Java assignments.
+
+Getting Started
+---------------
+
+Let's set up the assignment on our computer and learn about its basic structure.
+
+First, find Assignment #0 in Canvas and click on the invitation link. After accepting, go to your assignment repository on GitHub.
+
+Follow the instructions to :ref:`clone your repository <clone-csharp-project>` and open it in Visual Studio.
+
+When the solution opens, check out what the repository contains in the *Solution Explorer* on the left. You will find two projects within the solution.
+
+Open up the ``SayHelloClass`` class in the ``HelloWorldAutograded`` project. This is where you will work on your code.
+
+Now, turn your attention to ``Program.cs`` in the ``HelloWorldAutograded`` project. We have given you some code that will output the string returned by the ``SayHello()`` function in ``SayHelloClass``.
+
+To run the program, hit the *Run* button at the top right-hand corner of the window or right-click on the ``HelloWorldAutograded`` project and select *Run Project*.
+When the program runs, Visual Studio will open a pane at the bottom of the window and display program output. Currently, the program has no output. You'll fix that in a bit.
+
+Running the Autograding Script
+------------------------------
+
+For this unit, we encourage you to run the autograding tests yourself *before* you push your code.
+This will make it quicker and easier to know whether or not your code is correct. Let's learn how to run the tests.
+
+In the *Solution Explorer*, navigate to the test class, ``AutogradingTests``, in the project ``HelloWorldTests``.
+For each assignment, there will be a test project, which contains all of the autograding tests and may have more than one class.
+
+A test is a single method in a test file that has ``[TestMethod]`` just above it. A test file may have multiple tests. To run just one test, you can right-click on the test method name and select *Run Test*.
+To run all of the auto-grading tests (in this case, there is only one) but right-clicking on the project name and selecting *Run Project*.
```

Comment:
On the mac, I am not able to replicated running all the tests at once in this fashion unless I have previously tested.  If I select "Run" from VS's menu, I can then select "Run Unit Test".

Windows can do this using the right click method described in the directions no matter when i run tests.  (see note above).  Windows VS also has a "Test" menu tab where you can select "Run All Tests".

Reply:
Got it! Thanks!